Include/cpython/unicodeobject.h

```diff
@@ -132,21 +132,21 @@ typedef struct {
              * all characters are in the range U+0000-U+10FFFF
              * at least one character is in the range U+10000-U+10FFFF
          */
-        unsigned int kind:3;
+        uint16_t kind:3;
```

Comment:
These should remain `unsigned int`. `uint16_t` bitfield types are not standard compliant:

https://en.cppreference.com/w/c/language/bit_field#:~:text=Bit%2Dfields%20can%20have%20only%20one%20of%20these